### PR TITLE
feat: support trimming any text lines preceding the line containing the conventional commit type

### PR DESCRIPTION
**Pull request by albertocsm**
impl proposal for #415

### CODE REVIEW

**Comment by ga-paul-t** [comment]
@albertocsm I'll take a look at this asap. Really appreciate the contribution and it looks like a useful addition

**Comment by ga-paul-t** [comment]
@albertocsm, can you please rebase your branch? Thanks.

**Review by ga-paul-t — APPROVED** [review]

**Comment by albertocsm** [comment]
> @albertocsm, can you please rebase your branch? Thanks.

done

File: docs/static/schema.json [hunk 1]
```diff
@@ -190,9 +195,14 @@
...
         },
+        {
+          "required": [
+            "trimHeader"
+          ]
+        },
         {
           "skipPrerelease": [
-            "multiline"
+            "skipPrerelease"
```
Comment (albertocsm) [inline docs/static/schema.json:205]: this change is not related to this PR goal but it looked like a "typo". please confirm this is the case.

Comment (ga-paul-t) [reply docs/static/schema.json:205]: Great spot. This should be:

```json
{
  "required": [
     "skipPrerelease"
  ]
}
```

**Review by ga-paul-t — APPROVED** [review]

File: docs/static/schema.json (hunk 1 above)
Comment (ga-paul-t) [reply docs/static/schema.json:205]: I'll fix this shortly. The change looks great, and I don't want this to hold up merging this PR.

**Pull request merged**